packages/stark-ui/src/modules/collapsible/components/collapsible.component.spec.ts

```diff
@@ -0,0 +1,144 @@
+/*tslint:disable:completed-docs*/
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import {
+	STARK_LOGGING_SERVICE
+} from "@nationalbankbelgium/stark-core";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { StarkCollapsibleComponent } from "@nationalbankbelgium/stark-ui";
```

Comment:
This import is wrong, it should point to a relative path: the file where the component is declared